docs/metadata-distribution.md

```diff
@@ -2,7 +2,7 @@
 
 ?> Metadata is used to exchange information between Identity Provider and Service Provider. Simply there are two major ways to exchange Metadata.
 
-**Release in public **
```

Comment:
I happened to notice this while looking for contribution docs. Happy to extract it into another PR if appropriate.